cpp/lib/ome/xml/module.cpp

```diff
@@ -54,7 +54,7 @@ namespace
       ("ome-xml-data",
        "OME_XML_PKGDATADIR",
        "OME_XML_HOME",
-       "BIOFORMATS_HOME",
+       "OME_FILES_HOME",
```

Comment:
The use of `OME_FILES` here seems somewhat like a leaking concern if this will eventually migrate to `ome-model` (or similar), but that is possibly only temporary.